packages/skill-lesson/video/video-transcript.tsx

```diff
@@ -22,10 +22,15 @@ export const VideoTranscript: React.FC<{
 
   return (
     <div data-video-transcript="">
-      <h2 data-title="">Transcript</h2>
+      <h2 className="text-2xl font-bold" data-title="">
+        Transcript
+      </h2>
       <div data-transcript="">
         {typeof transcript === 'string' ? (
-          <ReactMarkdown components={markdownComponents}>
+          <ReactMarkdown
+            className="prose max-w-none pt-4"
+            components={markdownComponents}
```

Comment:
these should be styled via `data-` attribute in external stylesheet.

Reply:
https://github.com/skillrecordings/products/blob/main/packages/create-skill-app/templates/next/src/styles/globals.css#L9-L16